src/server/data/users/123456789/publications.json

```diff
@@ -0,0 +1,36 @@
+[
```

Comment:
don't upload test files

Reply:
added to gitignore. they are gone from git